src/main/java/com/example/LoginServlet.java

```diff
@@ -10,14 +10,18 @@
 
 @WebServlet(name = "login", description = "Trivial login page", urlPatterns = "/login")
 public class LoginServlet extends HttpServlet {
+
   @Override
   protected void doGet(HttpServletRequest req, HttpServletResponse res) throws IOException {
-    String redirectURL = req.getParameter("responseurl");
+
+    String redirectURL = req.getParameter("responseurl") + "&code=xxxxxx";
```

Comment:
If the redirect URL is encoded properly, then it should be parsed correctly without the extra `code` part. Is there an encoding issue happening?

Reply:
Yes, for me there is an encoding issue, the second parameter never comes, so that is why (as workaround) I changed the parameters order and then added code in line 17.

Reply:
I also opened the issue number #7 for it.